### PR TITLE
[NEW] Make the header for rooms clickable

### LINKED ISSUE #16223: Make the room name clickable and open room info
### Description:

Currently, the room name is not clickable.  

### Steps to reproduce:

1. Click on room name at the top left
2. It is not clickable

### Expected behavior:

It should be clickable and open the room info in the tab bar

### Server Setup Information:

- Version of Rocket.Chat Server: 2.5.0 -develop
- Operating System: ubuntu

### Additional context
I am working on it

### PR DESCRIPTION
**Pull request by aKn1ghtOut**
Closes #16223 

Earlier the title of the rooms wasn't clickable. An event was added to open the Room Info/User Info page when clicked, instead.

Earlier: Title not clickable

Now:
ezgif-3-b6c7467a9e26
ezgif-3-7e02d21f99d5

### CODE REVIEW

**Comment by aKn1ghtOut** [comment]
What do you think? @gabriellsh

**Comment by aKn1ghtOut** [comment]
@ggazzo What do you think?

**Comment by aKn1ghtOut** [comment]
@gabriellsh @rodrigok Anything I should add/edit?

**Comment by engelgabriel** [comment]
I think it makes sense

File: app/ui/client/components/header/headerRoom.js [hunk 1]
```diff
@@ -177,6 +177,31 @@ Template.headerRoom.events({
 			});
 		}
 	},
+	'click .rc-header__content.rc-header__block'(event, instance) {
+		const { tabBar } = instance.parentTemplate();
+		const $flexTab = $('.flex-tab-container .flex-tab');
+
+		if (tabBar.getState() === 'opened' && (tabBar.getTemplate() === 'channelSettings' || tabBar.getTemplate() === 'membersList')) {
+			$flexTab.attr('template', '');
```
Comment (rodrigok) [inline app/ui/client/components/header/headerRoom.js:185]: Is this right setting the html property manually?

Comment (gabriellsh) [reply app/ui/client/components/header/headerRoom.js:185]: It's been done the same way in flexTabBar.js. I think he got the snippet from there actually.

Comment (aKn1ghtOut) [reply app/ui/client/components/header/headerRoom.js:185]: That accurate. I was looking for how it had been implemented, and that's the file I got to.

**Pull request merged**